### PR TITLE
Import Documents: add options for meta matching in CoNLL-U

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

CoNLL-U files were only matched by "Text_ID". However, sometimes the data should be matched by "ID".

##### Description of changes
This is only a workaround, until we figure out how could the user define the proper column for joining metas to corpus.
Please, can someone comment on the tests?

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by markotoplak** [comment]
For failing tests I see this as one of the causes. I'll open a separate PR for it: biolab/orange3#6943

File: orangecontrib/text/import_documents.py [hunk 1]
```diff
@@ -513,13 +513,17 @@ def _add_metadata(self, corpus: Corpus) -> Corpus:
...
+                         self.CONLLU_META_DATA)
             )
         ):
             return corpus
 
         if self.is_conllu:
-            df = self._meta_data.set_index(self.CONLLU_META_DATA)
+            # find the first matching column
+            match_id = next((idx for idx in self.CONLLU_META_DATA if idx in self._meta_data.columns),
+                            None)
```
Comment (markotoplak) [inline orangecontrib/text/import_documents.py:525]: Why does this next() have a default value? Isn't the upper condition supposed to filter out those cases? If so, a default value here would only hide potential issues.

Comment (ajdapretnar) [reply orangecontrib/text/import_documents.py]: Fixed.

**Pull request merged**